deploy/conf/production-gs/site.conf

```diff
@@ -0,0 +1,48 @@
+#
+# DO NOT EDIT THIS FILE DIRECTLY, edit in https://github.com/gemini-hlsw/navigate-server/tree/main/deploy/conf/production-gs/site.conf
+
+mode = production
+site = GS
+
+# Web server related configuration
+web-server {
+    # External url used for redirects
+    external-base-url = "navigate.cl.gemini.edu"
+    # Uri to forward requests made to /proxy to.
+    proxy-base-uri = "http://localhost:4000"
+    # Certificate
+    tls {
+        key-store = "conf/local/site.jks"
+        key-store-pwd = "<Define in secrets.conf>"
+        cert-pwd = "<Define in secrets.conf>"
+    }
+}
+
+# Single sign-on configuration
+lucuma-sso {
+    service-token = "<Define in secrets.conf>"
+    sso-url = "https://sso.gpp.gemini.edu" # Production SSO Server and its public key
+    public-key = "-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nmQINBGVnsbcBEADx0D4eJrk/CdrpSvCmymQiTHDzNuKMw+Zt/HGct7T3VcLMKspz\ndaT76hSfK0tc1W4pEGANlX2CITYRdGwT7QycFnw9oXJ3eXDY8OTsT3sRbvJxPIyf\nt4sVQ8lS9L118BW/WXh8unESy2oUD78mlYU8FH3RreifbXqqrZJKEE20WvAguqj4\nLjh0thVQGnO6r1P0Om1iyc24c1GWL3JfsMjzLuY9CiiOAsCREAksKbbRBKP+uhB5\nzskJqKFqfb/MPE+0CqBXQhrFPtZHyMhhoS1hFVcQ00pBxSUTSFbhzHV+xoVbIxd+\nWd6UVYaChUX8jFl6Ua1p1Y8CW6lt0pSsygDwaF6FE/Q86cGuBPrUHwrld6WYeJNT\nCsvQ/buI/YGRcTLMXwwLqVJLkvobkU6tcco3eXHpvOb0v4i4e8R/qam0SX2OhejQ\n3ZmrDe/Hy9RdcWaRwthkfCKcN8F4R4YkWZNiYkSLD73giW+b4HYuY2QjcZ1gD0zE\nefEN9slo1d/ZJNld1iP5TYNrRsind2JSyMI6KfoYQlBzJya21hRUUCNtN6+hpEfj\nInlmV5xcBREtCOk5b28opT1fJ1p5XTO+WzPdYWzE6ENsf/6yUW/mlfkiDETaG2d4\n+FyNHl/ctIbPMWDaNvuOyhAHD/xkdnHMjh2JbdkvHeSd8os2GsMDZc3MAQARAQAB\ntDFSb2IgTm9ycmlzIChHUFAgU1NPIEtleSkgPHJvYi5ub3JyaXNAbm9pcmxhYi5l\nZHU+iQJRBBMBCAA7FiEEcjXBw4ysY4l2Hcvq4dvzizqg+FEFAmVnsbcCGwMFCwkI\nBwICIgIGFQoJCAsCBBYCAwECHgcCF4AACgkQ4dvzizqg+FGu1BAAmYVmFGPKcb1+\nbVZd+JesgSyQS04M8dVgpQ2Y32RWQmptcJ5gsKc726qHwHrlwhfPxKDL8Jt5f3yi\nEy4DZ92qcCQfBcwYj/mdKwITHPEa+NNd2zqZNF/O/klbOSMzh5d1+6qR+Eaaz9+i\nynUGHjqTou9gVe0IAMvSL79w0OrS6tzwqQwykDkQftPVppsWh05UsGMCotwoItcs\ngBRD98ALHjbp/oL5BUFpZ11DR/bpmrPYMrm5kwWTgmbRoeANcuQGCgOj/V0hq82E\nw3GJNzqJQ+p04aPwNMSGvCbl51h+cFTGc7aN/9LTVZlamX3Xdn+7jt6WQbnKX3Lz\nA1xZAKcGRJMl7MJ+ssFGsAUcWDIu7sl+0WZ01fqEFf+qeJmC3Mvss7BM+snO6l5A\n3dfY/7AR0yYxqvlLBFr8u/E06cVPuULqixFtheohyadIkl0L/wvEEGes9R/YvR/q\npnq3kqHzxijJLXMRw+eVa5Y0p20OsKWoz0VHaYe796gNyJqfS+uUCVE2SqxXvG7Q\n3ftJR87KxxeQk06HPt+JXsNzE4Gwj0wt6XMCJVQyjhjHtcTyxw1/MmpeGx5RSIRL\nTLY/xVcF8uCG3fVsEy5fAoNmzobbzuFc8mk9li0/VFjYmEzxH8dUp+JJXVKAihws\niBoo44Sfmg33U4VSmb9glM0/4b7aTiQ=\n=36lG\n-----END PGP PUBLIC KEY BLOCK-----"
+}
+
+# Configuration of the navigate engine
+navigate-engine {
+    # host for the odb
+    odb = "https://lucuma-postgres-odb-production.herokuapp.com/odb"
+    # host for observe
+    observe = "https://observe.cl.gemini.edu"
+    # system-control {
+    #     altair = simulated
+    #     gems = simulated
+    #     gcal = full
+    #     gpi = simulated
+    #     gsaoi = simulated
+    #     tcs = full
+    #     observe = full
+    # }
```

Comment:
Why are these parameters commented?

Reply:
Because I don't know what the correct configuration for production is. We don't have a production deployment yet from where I could copy it. Is it the same as for staging?

I also noticed that in our current staging configurations GN has `gcal = full` but GS has `gcal = simulated`, is that correct?

Reply:
I have to wait for IT to create the `noirlab/gpp-navigate-server` repo before merging.